dangerzone/podman/machine.py

```diff
@@ -81,6 +81,9 @@ def init(
             logger.info(f"Podman machine '{self.name}' already exists.")
             return
 
+        if platform.system() == "Windows":
+            container_utils.subprocess_run(["wsl", "--update"], check=False)
```

Comment:
Adding a comment would help. I suggest:
```suggestion
        # `check=False` is used here as we upgrade wsl in an opportunist way:
        # In case of failures, continue the normal flow.
        if platform.system() == "Windows":
            container_utils.subprocess_run(["wsl", "--update"], check=False)
```

Also, I was thinking about logging, but… it's already the case with our redirect output code, and so not needed here!